### DISCUSSION TITLE
Equipment Inspector not working?

### DISCUSSION THREAD

**Discussion by Lineae15** [runelite/runelite#17505] [Q&A] [answered]
Title says it all. Even when I choose to hold shift. I've tried uninstalling and re-installing it and it still doesn't work? Any suggestions?

**Comment by YvesW** [comment, marked as answer]
Generally pluginhub problems should be addressed at the author since we don't make, maintain or quality control hub plugins. In this case you are likely using too many (hub) plugins. Iirc there's a limitation of 7 player menu options max.
